lib/case-status.js

```diff
@@ -16,7 +16,7 @@ module.exports = (receiptNum) => {
       timeout: 45000
     }, (err, res, body) => {
       if (err)
-        return resolve(`!!${err.message}!!`)
+        return reject(err)
```

Comment:
If you want to get the error in the catch ```catch ```` 

an error should be thrown ``` throw(`!!${err.message}!!`) ```

I can write tests to exemplify.